README.md

```diff
@@ -74,6 +91,24 @@ Check that all TODO comments follow the same pattern and link a Jira ticket: `TO
 
 Check for non existing and duplicate paths in `.pre-commit-config.yaml`. Background: In a big codebase, the exclude lists can be quite long and it's easy to make a typo or forget to remove an entry when it's no longer needed.
 
+### `check-ownership`
```

Comment:
Same comments as in the hook description. Is there a way of de-duplicating this? Just referencing from one place to the other?

Reply:
Even better, we have [this hook](https://github.com/luminartech/dev-tools/blob/cbbc81f6e0c93ff1d4db81c8f03707fd3f0d17c8/.pre-commit-config.yaml#L19) which autogenerates the readme from the hook description.